resources/assets/less/bem/beatmap-discussion-editor-insertion-menu.less

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.beatmap-discussion-editor-insertion-menu {
```

Comment:
Um this is still not fixed. Either disable the editor on mobile or figure out how to do it on mobile (if even possible considering this new one rather heavily depends on mouse hover).

Also it's kind of a regression because the existing menu is (relatively) usable in mobile.